controller/src/beerocks/master/db/db.cpp

```diff
@@ -3137,6 +3386,22 @@ std::shared_ptr<node> db::get_node(sMacAddr al_mac, sMacAddr ruid)
     return get_node(key);
 }
 
+std::shared_ptr<node::radio> db::get_hostap_by_mac(const std::string &mac)
```

Comment:
Please change to
`std::shared_ptr<node::radio> db::get_hostap_by_mac(const sMacAddr &mac)`

No new code will be merged which does all the string to mac and mac to string conversions now that we have all that we need as part of the sMacAddr structure.

Reply:
done